subprojects/language-native/src/main/java/org/gradle/language/swift/SwiftComponent.java

```diff
@@ -69,4 +69,11 @@
      * Returns the implementation dependencies of this component.
      */
     Configuration getImplementationDependencies();
+
+    /**
+     * Returns the Swift language version supported of this component.
```

Comment:
We should make it clear that this is the version of the Swift _language_ and (default) version of the standard libraries that the source will be compiled against, and is not the ABI or target runtime version. So, we want a property name that communicates this and also a javadoc comment that makes this clear. "swiftLanguageVersionSupport" doesn't really capture this.

Reply:
I will think about how to better improve this.

Reply:
We renamed the property to `sourceCompatibility` to be more aligned with Java.